### PR TITLE
Copy button added to code snippets

### PR DESCRIPTION
**Pull request by softwarecurator**
- Added button to where user can copy any code snippet on markdown
snippet

- Also displays working button on editor screen
Screen Shot 2020-01-05 at 11 52 16 PM

Deployed link: https://manly-koi-29.reshuffle.app/ / checked styling on IPhone X screen size.

### CODE REVIEW

**Comment by rylandg** [comment]
@josephriosIO Does the link reflect the updates?

File: src/containers/ContentContainer.jsx [hunk 1]
```diff
@@ -23,6 +24,26 @@ const ContentContainer = function ({ html }) {
   // is always highlighted even after updates
   useEffect(() => {
     Prism.highlightAll();
+
+    const pres = document.getElementsByTagName("pre");
```
Comment (rylandg) [inline src/containers/ContentContainer.jsx]: single quotes

File: src/containers/ContentContainer.jsx [hunk 2]
```diff
@@ -23,6 +24,26 @@ const ContentContainer = function ({ html }) {
...
+
+    if (pres !== null) {
+      for (let codeSnippet of pres) {
+        if (codeSnippet.className === ' language-js' || codeSnippet.className === ' language-jsx') {
+          codeSnippet.innerHTML = `<div class="copy"><i class="fas fa-copy"></i></div><code class="${codeSnippet.className}">${codeSnippet.innerHTML}</code>`;
+        }
+      }
+      const clipboard = new Clipboard('.copy', {
+        target: trigger => {
```
Comment (rylandg) [inline src/containers/ContentContainer.jsx]: This line could be better `target: ({ nextElementSibling }) => nextElementSibling`

File: src/style/oceanic.scss [hunk 3]
```diff
@@ -49,6 +43,35 @@ code[class*="language-"] ::selection {
...
+  right: -10px;
+  transform: translate(-10px, -90%);
+  border-top-left-radius: 5px;
+  border-top-right-radius: 5px;
+  color: white;
+  background: rgb(18,29,35);
+  background: linear-gradient(180deg, rgba(18,29,35,1) 61%, rgba(27,43,52,1) 100%);
+  cursor: pointer;
+  transition: all 200ms ease;
+}
```
Comment (rylandg) [inline src/style/oceanic.scss]: put newlines between classes

**Review by rylandg — APPROVED** [review]
Nitpicks

File: src/containers/ContentContainer.jsx (hunk 2 above)
Comment (josephriosIO) [reply src/containers/ContentContainer.jsx]: Will change!

**Pull request merged**